bouncer/commands/stop_boosting.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S pnpm tsx
+// INSTRUCTIONS
+//
+// This command takes 3 arguments:
+// 1 - Asset
+// 2 - Tier
+// 3 (optional) - Account URI (Default: "//LP_1")
```

Comment:
Maybe we should default to "//LP_2" since that's what we use for boost in general? or `LP_BOOST` ?